src/store-api/src/storage/requests.rs

```diff
@@ -58,6 +58,9 @@ pub struct ScanRequest {
     /// If set, only rows with a sequence number lesser or equal to this value
     /// will be returned.
     pub sequence: Option<SequenceNumber>,
```

Comment:
Do we need to rename this to max_sequence?

Reply:
> Do we need to rename this to max_sequence?

still useful for refilling which might still need to limit the sequence range and such

Reply:
Will renaming this field break this?